File: src/main/java/land/leets/domain/auth/presentation/dto/OAuthTokenDto.java
package land.leets.domain.auth.presentation.dto;

import lombok.*;

@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@Getter
public class OAuthTokenDto {
    private String access_token;
    private String expires_in;
    private String scope;
    private String token_type;
    private String id_token;
}

File: src/main/kotlin/land/leets/domain/auth/presentation/dto/OAuthTokenDto.kt
package land.leets.domain.auth.presentation.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class OAuthTokenDto(
    @JsonProperty("access_token") val accessToken: String,
    @JsonProperty("expires_in") val expiresIn: String,
    val scope: String,
    @JsonProperty("token_type") val tokenType: String,
    @JsonProperty("id_token") val idToken: String
)
